client/README.md

```diff
@@ -23,17 +21,16 @@ This extension provides Puppet Language support for [Visual Studio Code](https:/
 - Import from `puppet resource` directly into manifests
```

Comment:
Is it worth pointing out here that this is a "technical preview" (or other term)?  and that this release is mainly to gather bugs and feedback prior to a 1.0 release?